content/doc/developer/development-environment/index.adoc

```diff
@@ -1,24 +1,34 @@
 ---
 layout: developerchapter
-wip: true
 summary: Preparing your development environment for Jenkins core and plugin development
 references:
-- url: https://wiki.jenkins.io/display/JENKINS/IntelliJ+IDEA+plugin+for+Stapler
-  title: IntelliJ IDEA plugin for Stapler
-- url: https://wiki.jenkins.io/display/JENKINS/NetBeans+plugin+for+Stapler
-  title: NetBeans plugin for Stapler
-- url: https://wiki.jenkins.io/display/JENKINS/Setting+up+Eclipse+to+build+Jenkins
```

Comment:
All the eclipse guides seem ancient and referring to bugs in very old versions, seems more harmful than helpful to even mention them?

Reply:
If they're bad, clear the wiki pages 😃